unittests/Dialect/Moore/TypesTest.cpp

```diff
@@ -131,14 +131,14 @@ TEST(TypesTest, Structs) {
   auto bit8Type = IntType::getInt(&context, 8);
   auto bitDynArrayType = OpenUnpackedArrayType::get(bitType);
 
-  auto s0 = StructType::get(&context, {StructMember{foo, bitType}});
-  auto s1 = StructType::get(
-      &context, {StructMember{foo, bitType}, StructMember{bar, bit8Type}});
-  auto s2 = StructType::get(
-      &context, {StructMember{foo, bitType}, StructMember{bar, logicType}});
-  auto s3 =
-      UnpackedStructType::get(&context, {StructMember{foo, bitType},
-                                         StructMember{bar, bitDynArrayType}});
+  auto s0 = StructType::get(&context, {StructLikeMember{foo, bitType}});
+  auto s1 = StructType::get(&context, {StructLikeMember{foo, bitType},
+                                       StructLikeMember{bar, bit8Type}});
+  auto s2 = StructType::get(&context, {StructLikeMember{foo, bitType},
+                                       StructLikeMember{bar, logicType}});
+  auto s3 = UnpackedStructType::get(
+      &context,
+      {StructLikeMember{foo, bitType}, StructLikeMember{bar, bitDynArrayType}});
```

Comment:
Can you add a few checks with `UnionType` here and ensure that `.getBitSize()` returns the correct numbers for unions? I think in contrast to structs you would want to return the size of the largest member in the union. That's worth quickly checking here 😃

Reply:
Sorry for missing this part, code has been uploaded.
BTW, I found tagged keyword some unsupported case and submitted an issue with slang developer.
https://github.com/MikePopoloski/slang/issues/1012